### ISSUE TITLE
Question about bin size for TAD calling

### ISSUE THREAD

**Issue by nyuhic** [deeptools/HiCExplorer#373]
Hello,

Firstly, this is not an technical  issue but just a question I have to improve my own understanding. 

I have constructed a matrix at restriction fragment resolution (~800bp) and then merged bins to produce matrices with bin sized of ~ 10kb, 50kb , 100kb and 1MB. The diagnostic plots for these bin sizes are similar and look more or less normally distributed. For example. at 100kb bin size the plot looks like this:
![HET_Merged_rest h5 125binsmerged h5diagnostic_plot](https://user-images.githubusercontent.com/49555006/56267223-44e07b00-60ff-11e9-89fa-ee1ed45e7145.png)

The plot for the restriction fragment resolution is however very different (which I assume is to be expected)
![HET_Merged_rest h5diagnostic_plot](https://user-images.githubusercontent.com/49555006/56267269-6ccfde80-60ff-11e9-9bae-c149c7b742f1.png)

I understand that different analyses use different bin sizes. However, for TAD calling it is usually recommended to use the highest resolution matrix. So in this case is it  still a good idea to correct and use the restriction fragment resolution matrix for TAD calling given how different the distribution of bins is or is it better to use a merged matrix that has more normally distributed bin sizes like 10kb?

Thanks

**Comment by LeilyR** [comment]
In theory for a deep enough data one should be able to call TADs on the restriction fragment resolution, however in reality one might need to merge few bins to compensate for the data sparsity.  Sometime  it is a good practice if one tries to visualize TADs of several resolution and compare.  Your second plot shows how sparse your data, so you might need to consider merging some bins, however 10kb might be a bit too much, if you are looking for high resolution TADs. You could try merging less number bins and see how your matrix look for something like 2 or 3kb  resolution.

**Comment by nyuhic** [comment]
Thanks for the reply. I tried a number of different resolutions and plotted TADs. Looks like data is just not deep enough for 2 or 3kb resolution. My plots only looked like TADs at resolutions of a 8-10kb or above.

One more question I have is about normalization. If I want to compare TAD boundaries and scores among different conditions, should I normalized matrices before calling TADs? My understanding is that the hicnormalize function should be used to normalize matrices of different conditions before plotting them but I wasn't sue if the normalization would also effect TAD boundaries or scores?

Thanks
